frontend/src/v5/ui/components/shared/containerRevisionDetails/containerRevisionDetails.component.tsx

```diff
@@ -26,9 +26,9 @@ import { FormattedMessage } from 'react-intl';
 import { UploadStatus } from '@/v5/store/containers/containers.types';
 import { canUploadToBackend } from '@/v5/store/containers/containers.helpers';
 import { uploadToContainer } from '@/v5/ui/routes/dashboard/projects/containers/uploadContainerRevisionForm/uploadContainerRevisionForm.helpers';
-import { SkeletonListItem } from './components/skeletonListItem/skeletonListItem.component';
-import { RevisionsListItem } from './components/revisionsListItem/revisionsListItem.component';
-import { RevisionsListHeaderLabel } from './components/revisionsListHeaderLabel/revisionsListHeaderLabel.component';
+import { SkeletonListItem } from '../revisionDetails/components/skeletonListItem/skeletonListItem.component';
```

Comment:
Latest revision 
which has the format {statusCode}-{revisionCode}
<img width="992" alt="image" src="https://github.com/3drepo/3drepo.io/assets/42032433/a5f58452-66e5-4072-a795-838ba366486a">